File: Lista1/Python/ex_6.py
soma_pares = 0
soma_impares = 0
soma_div3 = 0
for c in range(1,11):
    numero = int(input(f'Digite o {c}° valor: '))
    if numero % 2 == 0:
        soma_pares += numero
    else:
        soma_impares += numero
    if numero % 3 == 0:
        soma_div3 += numero

print(f'Soma dos valores pares: {soma_pares}')
print(f'Soma dos valores impares: {soma_impares}')
print(f'Soma dos valores divisiveis por 3: {soma_div3}')

File: Lista1/Java/ex_6.java
import java.util.Scanner;
public class ex_6
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    int soma_pares = 0;
        int soma_impares = 0;
        int soma_div3 = 0;
	    for (int c = 1; c<=10; c++) {
	      System.out.println("Digite o " + c + "° numero: ");
	      int numero = sc.nextInt();
	      if (numero % 2 == 0) {
            soma_pares += numero;
          } 
          else {
            soma_impares += numero;
          }

          if (numero % 3 == 0) {
                soma_div3 += numero;
          }
	    }
		System.out.println("A soma dos valores pares somados é: " + soma_pares);
        System.out.println("A soma dos valores impares somados é: " + soma_impares);
        System.out.println("A soma dos valores divisiveis por 3 somados é: " + soma_div3);
	}
}